tool/tctl/common/access_request_command.go

```diff
@@ -147,7 +151,30 @@ func (c *AccessRequestCommand) TryRun(ctx context.Context, cmd string, client au
 }
 
 func (c *AccessRequestCommand) List(ctx context.Context, client auth.ClientI) error {
-	reqs, err := client.GetAccessRequests(ctx, types.AccessRequestFilter{})
+	var index proto.AccessRequestSort
+	switch c.sortIndex {
+	case "created":
+		index = proto.AccessRequestSort_CREATED
+	case "state":
+		index = proto.AccessRequestSort_STATE
+	default:
+		return trace.BadParameter("unsupported sort index %q (expected one of 'created' or 'state')", c.sortIndex)
+	}
```

Comment:
Deliberate omission since the `DEFAULT` (id) sort order isn't really intended to be user-facing.  I want the `DEFAULT` sort order to be implementation-defined rather than promising to always be ID, so that it can always be whatever the native/efficient ordering is.